_posts/2022-11-01-ndk_advanced_custom_fields.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title: "[CVE-2022-40839] Improper neutralization of SQL parameter in NdkAdvancedCustomizationFields module for PrestaShop"
+categories: module
+author:
+- Friends-Of-Presta.org
+meta: "CVE,PrestaShop,ndk_advanced_custom_fields"
+severity: "critical (9.8)"
+---
+
+A SQL injection vulnerability in the height and width parameter in NdkAdvancedCustomizationFields v3.5.0 allows unauthenticated attackers to exfiltrate database data.
+
+## Summary
+
+* **CVE ID**: [CVE-2022-40839](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-40839)
+* **Published at**: 2022-11-01
+* **Advisory source**: [@daaaalllii](https://github.com/daaaalllii/cve-s/blob/main/CVE-2022-40839/poc.txt)
+* **Vendor**: PrestaShop
+* **Product**: ndk_advanced_custom_fields
+* **Impacted release**: <= 3.5.0
+* **Product author**: ndk design
+* **Weakness**: [CWE-89](https://cwe.mitre.org/data/definitions/89.html)
+* **Severity**: critical (9.8)
+
+## Description
+
+## CVSS base metrics
+
+* **Attack vector**: network
+* **Attack complexity**: low
+* **Privilege required**: none
+* **User interaction**: none
+* **Scope**: unchanged
+* **Confidentiality**: high
+* **Integrity**: high
+* **Availability**: high
+
+**Vector string**: [CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H)
+
+## Possible malicious usage
+
+* Obtain admin access
+* Remove data from the associated PrestaShop
+* Copy/paste data from sensitive tables to FRONT to exposed tokens and unlock admins's ajax scripts
```

Comment:
Copy/paste data from sensitive tables to FRONT to expose tokens and unlock admins's ajax scripts